mentors/Makefile

```diff
@@ -0,0 +1,77 @@
+DOCKER_IMAGE?=mentor-processing-pipeline
+DOCKER_CONTAINER=mentor-processing-pipeline
+MENTOR?=julianne
+SESSION?=1
+WATSON_USERNAME=`head -n 1 secrets/watson_credentials.txt`
```

Comment:
I think these will raise errors if the files don't exist. Maybe could be

```
WATSON_USERNAME=$(shell if [ -f secrets/watson_credentials.txt ]; then head -n 1 secrets/watson_credentials.txt; else echo ""; fi)
```

...might require you to explicitly set the shell to bash above with

```
SHELL:=/bin/bash
```

...but we do that in many Makefiles. I suspect this one is not windows compatible anyway

Reply:
If the files are missing don't we want to raise the errors?

Reply:
I think log an error, but the error you'll get from just reading an file that doesn't exist will be confusing, and also, not every make rule will require watson creds.

Another thing I forgot is that the `WATSON_XXX` vars should only look for the files if the env var is not already set, so all in all, maybe this:

```
WATSON_USERNAME?=$(shell if [ -f secrets/watson_credentials.txt ]; then head -n 1 secrets/watson_credentials.txt; else echo ""; fi)
```

```
.PHONY: watson-credentials-set
watson-credentials-set:
ifeq ("$(WATSON_USERNAME)", "")
	@echo "WATSON_USERNAME must be set in the env or in ./secrets/watson-credentials.txt"
	exit 1
endif

.PHONY: build-audiochunks
build-audiochunks: watson-credentials-set
	docker run \
			-it \
			--rm \
			--name $(DOCKER_CONTAINER) \
			-e WATSON_USERNAME=$(WATSON_USERNAME) \
			-e WATSON_PASSWORD=$(WATSON_PASSWORD) \
			-v $(shell pwd)/recordings:/app/recordings \
			$(DOCKER_IMAGE) -m $(MENTOR) -u $(URL)
```